tests/java/PMDBearTest.py

```diff
@@ -34,7 +34,7 @@ class Hello {
 """.splitlines(keepends=True)
 
 
-InferBearTest = verify_local_bear(PMDBear,
-                                  valid_files=(good_file,),
-                                  invalid_files=(bad_file,),
-                                  tempfile_kwargs={"suffix": ".java"})
```

Comment:
you've done the changes in the wrong commit ... to edit another commit, use ``git rebase -i master`` and edit that one, we cannot have this change in this commit, it's hard to keep track later on.